### ISSUE TITLE
Merge JIC traits with master.

### ISSUE THREAD

**Issue by cpommier** [Planteome/CO_321-wheat-traits#17]
There has been version mixing. JIC variables have been added to the www.cropontology.org website without reusing the latest version which was available only in github.
Action:
* JIC version downloaded from Crop ontology web site and saved in the github as CO_321.csv for archive
* Add JIC variables at the end of the master version
* pull request

**Comment by cpommier** [comment]
**log**:
* get files
```
create new Branch : fix/mergeJIC
Import CO_321.csv from crop Ontology in Archive
```
* order files content
```
Open with LibreOffice to preserve quoted CSV 
Reorder by variableID. 
Save as CSV. Note that empty fields are not quoted anymore
CO_321.csv
wheatTD.csv
```

**Comment by cpommier** [comment]
JIC Variables added:
```
  | CO_321:0001700 | GNitr_NIRS_pct | GRN_pct | Research-intensive characterization | After harvest |   | JIC | Luzie U Wingen | 24-juil-20 | EN | Wheat | CO_321:1010105 | Grain Nitrogen content | Quality traits | Whole-grain   Nitrogen content. | GNitr | GNC | Grain | Nitrogen   content |   | CO_321:0000946 | NIRS Measurement | Measurement | NIR calibrated   based on Kjeldahl results. | CO_321:0000410 | % | Numerical |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |  
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
  | CO_321:0001701 | STRNitr_NIRS_pct | Research-intensive characterization | After harvest |   | JIC | Luzie U Wingen | 24-juil-20 | EN | Wheat | CO_321:1010043 | Straw Nitrogen content | Physiological traits | Nitrogen content of the above ground   straw biomass. | STRNitr | StrawN | Straw | Nitrogen content |   | CO_321:0001678 | STR NIRS Measurement | Measurement | Straw NIR calibrated. |   | CO_321:0000410 | % | Numerical |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |  
  | CO_321:0001702 | AGBMNitr_M_pct | Research-intensive characterization | After harvest |   | JIC | Luzie U Wingen | 24-juil-20 | EN | Wheat | CO_321:0001662 | Above ground biomass nitrogen content | Physiological traits | Nitrogen content of the above ground   biomass. | BMNitr | AGN | Plant | Nitrogen content |   | CO_321:0001682 | BM NIRS Computation | Computation | Calculated as sum of straw Nitrogen   content and grain Nitrogen content according to the biomass partition into   straw and grain. | CO_321:0000410 | % | Numerical |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |  
  | CO_321:0001703 | NHI_Calc_pct | Nitrogen harvest index | Research-intensive characterization | After harvest |   | JIC | Luzie U Wingen | 24-juil-20 | EN | Wheat | CO_321:0001666 | Nitrogen harvest index | Physiological traits | The fraction of Nitrogen recovered   in the harvested grain as a function of the total Nitrogen taken up. | NHI |   | Plant | Nitrogen harvest index |   | CO_321:0001686 | NHI Computation | Computation | Calculated as total grain Nitrogen divided by the total biomass Nitrogen. | NHI= grain Nitrogen / above ground biomass Nitrogen | Cormier et al.,2016 | CO_321:0000410 | % | Numerical |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |  
  | CO_321:0001704 | GFibre_NIRS_pct | GRNFibre_pct | Trial evaluation | After harvest (From DS87) | JIC | Luzie U Wingen | 24-juil-20 | EN | Wheat | CO_321:0001673 | Grain fibre content | Quality traits | Content of fibre (corrected to 12.5%   moisture basis) in the whole wheat grain samples. | Gfibre | GRNFibre | Grain | Fibre content |   | CO_321:0001693 | NIRS Fibre Measurement | Measurement | NIR calibrated |   | CO_321:0000410 | % | Numerical |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |  
  | CO_321:0001705 | GStarch_NIRS_pct | GRNStarch_pct | Trial evaluation | After harvest (From DS87) | JIC | Luzie U Wingen | 24-juil-20 | EN | Wheat | CO_321:0001674 | Grain starch content | Quality traits | Content of starch (corrected to   12.5% moisture basis) in the whole wheat grain samples. | Gstarch | GRNSTARCH | Grain | Starch content |   | CO_321:0001695 | NIRS Starch Measurement | Measurement | NIR calibrated |   | CO_321:0000410 | % | Numerical |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |  
  | CO_321:0001706 | GPD_Calc_pct | Research-intensive characterization | After harvest |   | JIC | Luzie U Wingen | 24-juil-20 | EN | Wheat | CO_321:0001672 | Grain protein deviation | Physiological traits | Deviation from the grain yield   -grain protein linear regression. | GPD |   | Plant | Grain Protein Deviation |   | CO_321:0001692 | GPD Computation | Computation | Calculated as the deviation from the   yield-protein linear regression. | Cormier et al.,2016 | CO_321:0000410 | % | Numerical |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |  
  | CO_321:0001707 | StrawYld_Calc_tha | Research-intensive characterization | After harvest |   | JIC | Luzie U Wingen | 24-juil-20 | EN | Wheat | CO_321:0000977 | Straw yield | Agronomical traits | Amount (weight) of straw. | StrawYld | STRBM, StrawDW | Straw | Yield |   |   | CO_321:0001677 | StrawYield_Computation | Computation | Calculated as difference between   biomass and grain yield. | Bell and Fischer, 1994, Reynolds et al., 2001, Pask et al., 2012 | CO_321:0000432 | t/ha | Numerical | 2.0 | 0.0 | 20.0 |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |  
  | CO_321:0001708 | Awns_E_0to1 | AWNS_0_1 | Descriptor | DS65-DS89 |   |   | JIC | Luzie U Wingen | 24-juil-20 | EN | Wheat | CO_321:0000027 | Spike awnedness | Morphological traits | Observation of awn type whether   awnless/awnletted (short awns)/awned (conspicious awns). | Awns |   | Spike | Awnedness |   |   | CO_321:0001676 | Awns Presence | Estimation | Standard method for awn presence   estimation. | CO_321:0000599 | 0-1 absent-present | Nominal |   |   |   |   | 0= absent | 1= present |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |  
  | CO_321:0001709 | GNUP_Calc_kgha | Research-intensive characterization | After harvest |   | JIC | Luzie U Wingen | 24-juil-20 | EN | Wheat | CO_321:0001663 | Grain Nitrogen uptake | Physiological traits | Uptake of Nitrogen in the whole   wheat grain samples. | GNUp |   | Grain | Nitrogen uptake |   | CO_321:0001683 | GNUp Computation | Computation | Calculated as total amount of   Nitrogen in the grain at harvest. | CO_321:0000804 | kg/ha | Numerical |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |  
  | CO_321:0001710 | STRNUP_Calc_kgha | Research-intensive characterization | After harvest |   | JIC | Luzie U Wingen | 24-juil-20 | EN | Wheat | CO_321:0001665 | Straw Nitrogen uptake | Physiological traits | Uptake of Nitrogen in the whole   above ground straw. | STRNUp |   | Straw | Nitrogen uptake |   | CO_321:0001685 | STRNUp Computation | Computation | Calculated as total amount of   Nitrogen in the straw at harvest. | CO_321:0000804 | kg/ha | Numerical |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |  
  | CO_321:0001711 | BMNUP_Calc_kgha | Research-intensive characterization | After harvest |   | JIC | Luzie U Wingen | 24-juil-20 | EN | Wheat | CO_321:0001664 | Above ground biomass nitrogen uptake | Physiological traits | Uptake of Nitrogen in the above   ground biomass at maturity. | BMNUp |   | Plant | Nitrogen uptake |   | CO_321:0001684 | BMNUp Computation | Computation | Calculated as total amount of   Nitrogen in the above-ground biomass at harvest. | CO_321:0000804 | kg/ha | Numerical |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |  
  | CO_321:0001712 | GFPTT_Calc_Cday | GFILLPERIOD_thermal_time | Research-intensive characterization | DS65-DS87 |   |   | JIC | Luzie U Wingen | 24-juil-20 | EN | Wheat | CO_321:0001659 | Grain filling period thermal time | Phenological traits | Time period between anthesis (ds65)   and maturity (ds87) in thermal time. | GFPTT | GfillPeriodTT | Plant | Grain filling period thermal time | CO_321:0001679 | GFPTT Computation | Computation | Calculated  with climatic data where degree days are   accumulating between anthesis (DS65) and maturity (DS87) of crop cycle   duration (thermal time from day1+day2+day3...). To calculate daily thermal   time= (Temperature max + Temperature min)/2. | CO_321:0001587 | degree C day | Numerical | 1.0 | 0.0 | 2000.0 |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |  
  | CO_321:0001713 | GsurfA_M_mm2 | GRSA_M_mm2 | Trial evaluation | After harvest (From DS87) | JIC | Luzie U Wingen | 24-juil-20 | EN | Wheat | CO_321:0001675 | Grain surface area | Morphological traits | Surface of whole grain. | GsurfA | GRSA | Grain | Surface area |   | CO_321:0001694 | GSurfA_Measurement | Measurement | Marvin method for whole surface   area. | https://www.marvitech.de/en/products-for-seed-analysis/ | CO_321:0001696 | mm2 | Numerical |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |  
  | CO_321:0001714 | GFR_Calc_gday | GFILLRATE | Research-intensive characterization | DS65-DS87 |   |   | JIC | Luzie U Wingen | 24-juil-20 | EN | Wheat | CO_321:0001660 | Grain fill rate | Phenological traits | Grain weight increase per day in   the  period between anthesis (ds65) and   maturity (ds87). | GFR | GfillRate | Plant | Grain fill rate |   | CO_321:0001680 | GFR Computation | Computation | Calculated daily grain weight   increase between anthesis (DS65) and maturity (DS87) of crop cycle duration. | CO_321:0001697 | g/day | Numerical |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |  
  | CO_321:0001715 | GFRTT_Calc_gCday | GFILLRATE_thermal_time | Research-intensive characterization | DS65-DS87 |   |   | JIC | Luzie U Wingen | 24-juil-20 | EN | Wheat | CO_321:0001661 | Grain fill rate thermal time | Phenological traits | Grain weight increase per degree day   in the  period between anthesis (ds65)   and maturity (ds87). | GFRTT | GfillRateTT | Plant | Grain fill rate thermal time | CO_321:0001681 | GFRTT Computation | Computation | Calculated  grain weight increase per degree day with   climatic data where degree days are accumulating between anthesis (DS65) and   maturity (DS87) of crop cycle duration (thermal time from day1+day2+day3...).   To calculate daily thermal time= (Temperature max + Temperature min)/2. | CO_321:0001698 | g/degree C day | Numerical |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |  
  | CO_321:0001716 | NupE_Calc_kgkgN | Research-intensive characterization | After harvest |   | JIC | Luzie U Wingen | 24-juil-20 | EN | Wheat | CO_321:0001667 | Nitrogen uptake efficiency | Physiological traits | N-uptake efficiency as the total   amount of Nitrogen in the above-ground plant at harvest in relation to the   available Nitrogen in soil. | BMNUpE | NupE | Plant | Nitrogen Uptake Efficiency | CO_321:0001687 | NupE Computation | Computation | Calculated as above ground biomass Nitrogen content divided by soil   Nitrogen | NUpE= above ground biomass Nitrogen / soil Nitrogen | Cormier et al.,2016 | CO_321:0001699 | kg/kg N | Numerical |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |  
  | CO_321:0001717 | NutE_Calc_kgkgN | Nitrogen utilisation efficiency | Research-intensive characterization | After harvest |   | JIC | Luzie U Wingen | 24-juil-20 | EN | Wheat | CO_321:0001668 | Grain Nitrogen utilization efficiency | Physiological traits | Nitrogen utilization efficiency as   the grain dry mass in relation to the total amount of Nitrogen in   above-ground plant at harvest. | NutE |   | Grain | Nitrogen Utilization Efficiency | CO_321:0001688 | Nute Computation | Computation | Calculated as grain yield per unit of Nitrogen taken up. | NutE= grain yield / above ground Nitrogen | Cormier et al.,2016 | CO_321:0001699 | kg/kg N | Numerical |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |  
  | CO_321:0001718 | BMNUtE_Calc_kgkgN | Research-intensive characterization | After harvest |   | JIC | Luzie U Wingen | 24-juil-20 | EN | Wheat | CO_321:0001669 | Biomass Nitrogen utilization efficiency | Physiological traits | Biomass Nitrogen Utilization   Efficiency as the biomass in relation to the total amount of N in   above-ground plant at harvest. | BMNUtE |   | Plant | Nitrogen Utilization Efficiency | CO_321:0001689 | BMNUte Computation | Computation | Calculated as biomass per unit of Nitrogen taken up. | BMNUtE= above ground biomass / above   ground Nitrogen | CO_321:0001699 | kg/kg N | Numerical |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |  
  | CO_321:0001719 | NUE_Calc_kgkgN | Nitrogen use efficiency | Research-intensive characterization | After harvest |   | JIC | Luzie U Wingen | 24-juil-20 | EN | Wheat | CO_321:0001670 | Nitrogen use efficiency | Physiological traits | The grain dry mass (grain yield)   production per  Nitrogen available  to a plant. | NUE |   | Grain | Nitrogen Use Efficiency |   | CO_321:0001690 | NUE Computation | Computation | Calculated as the grain yield divided by the total soil Nitrogen. | NUE =  Grain Yield at Maturity /   Soil Nitrogen | Cormier et al.,2016 | CO_321:0001699 | kg/kg N | Numerical |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |  
  | CO_321:0001720 | BMNUE_Calc_kgkgN | Research-intensive characterization | After harvest |   | JIC | Luzie U Wingen | 24-juil-20 | EN | Wheat | CO_321:0001671 | Above ground biomass nitrogen efficiency | Physiological traits | The above ground biomass production   per total Nitrogen available  to a   plant. | BMNUE |   | Plant | Nitrogen Use Efficiency |   | CO_321:0001691 | BMNUE Computation | Computation | Calculated as the above ground biomass divided by the total soil   Nitrogen. | BMNUE =  Above Ground Biomass at   Maturity / Soil Nitrogen | Cormier et al.,2016 | CO_321:0001699 | kg/kg N | Numerical |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |  

```
